src/test/resources/mock.web-api-server.core.1.0.2.resoscript

```diff
@@ -511,84 +498,6 @@
-    />
-
-    <Request
-      RequestId="REQ-WA103-QM5.2"
-      OutputFile="REQ-WA103-QM5_GeoSpatialLongLat_LatGT1_PointGeo.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_GeoSpatialField*&amp;$filter=geo.distance(*Parameter_GeoSpatialField*, geography'SRID=*Parameter_SRID*;POINT(*Parameter_GeoSpatialValue*)' le *Parameter_GeoSpatialDistanceValue* and *Parameter_GeoSpatialLatitudeField* gt 1"
-    />
-
-    <Request
-      RequestId="REQ-WA103-QO12"
-      OutputFile="REQ-WA103-QO12_Grouping.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_IntegerField*&amp;$filter=(*Parameter_IntegerField* ge *Parameter_IntegerValueLow* and *Parameter_IntegerField* le *Parameter_IntegerValueHigh*) and (*Parameter_IntegerField* lt *Parameter_IntegerValueHigh* and *Parameter_IntegerField* gt *Parameter_IntegerValueLow*)*Parameter_RequiredParameters*"
-    />
-
-    <Request
-      RequestId="REQ-WA103-QO13"
-      OutputFile="REQ-WA103-QO13.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_StringField*&amp;$filter=contains(*Parameter_StringField*,'*Parameter_ContainsValue*')*Parameter_RequiredParameters*"
-    />
-
-    <Request
-      RequestId="REQ-WA103-QO14"
-      OutputFile="REQ-WA103-QO14_EndsWith.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_StringField*&amp;$filter=endswith(*Parameter_StringField*,'*Parameter_EndsWithValue*')*Parameter_RequiredParameters*"
-    />
-
-    <Request
-      RequestId="REQ-WA103-QO15"
-      OutputFile="REQ-WA103-QO15_StartsWith.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_StringField*&amp;$filter=startswith(*Parameter_StringField*,'*Parameter_StartsWithValue*')*Parameter_RequiredParameters*"
-    />
-
-    <Request
-      RequestId="REQ-WA103-QO16"
-      OutputFile="REQ-WA103-QO16_ToLowerEQ.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_StringField*&amp;$filter=tolower(*Parameter_StringField*) eq '*Parameter_ToLowerValue*'*Parameter_RequiredParameters*"
-    />
-
-    <Request
-      RequestId="REQ-WA103-QO17"
-      OutputFile="REQ-WA103-QO17_ToUpperEQ.json"
-      Url="*ClientSettings_WebAPIURI*/*Parameter_EndpointResource*?$top=*Parameter_TopCount*&amp;$select=*Parameter_KeyOrKeyNumericField*,*Parameter_StringField*&amp;$filter=toupper(*Parameter_StringField*) eq '*Parameter_ToUpperValue*'*Parameter_RequiredParameters*"
-    />
-

```

Comment:
Even though these aren't part of core, is there a good place to keep them for when they get used by endorsements? Or are they already somewhere else in the codebase

Reply:
I figured we'd just go back in GitHub and grab them when we needed them. They will have their own `.resoscript` files and acceptance test `.feature` files. I'll create an issue so as not to forget.

Reply:
Issue #33 created.